### DISCUSSION TITLE
Ip Pools Not Showing in Ui After Enabling Ip Pools True

### DISCUSSION THREAD

**Discussion by mkmahadirdp** [postalserver/postal#3045] [Help with using Postal]
i have used this perfectly to enable Ip pools but Ip pools not showing in ui 

in postal.yml
i have configured like this 

version: 2

postal:
  web_hostname: mail.mydomain.com
  web_protocol: https
  smtp_hostname: mail.mydomain.com
  
web:
  host: postal.mydomain.com
  protocol: https 

general:
  use_ip_pools: true

web_server:
  # Specify configuration for the Postal web server
  bind_address: 154.173.111.34
  port: 5000

smtp_server:
  # Specify configuration to the Postal SMTP server
  port: 25

logging:
  # Specify options for the logging
  stdout: true

used commands 

postal stop
postal start
postal restart
postal upgrade

postal status is ok there is no problem

what can i do now?
thanks,

**Comment by ghostverse** [comment]
postal:
  web_hostname: postal.yourdomain.com
  web_protocol: https
  smtp_hostname: postal.yourdomain.com
  use_ip_pools: true  # Enable IP pools

After saving your config file, you can restart Postal to make it take effect. 

postal restart

**Comment by willpower232** [comment]
It sounds like they should be enabled so the only thing I can imagine is that the config file isn't making its way into your container somehow.

If you have a look at `postal logs app` it would say somewhere at the top (shortly after being restarted) about whether it found a config file or not.

You could also try `postal bash app`  and then `cd /config` to see the file for yourself.
